.blinkmrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "server": {
+    "project": "bm-service-settings.blinkm.io",
+    "region": "ap-southeast-2"
```

Comment:
I initially thought it would be weird to check these details in, but they'll be present in all of our other CLIs anyhow so I don't think there's a reasonable alternative.